### ISSUE TITLE
Enable pretendToBeVisual by default

### ISSUE THREAD

**Issue by just-boris** [modosc/global-jsdom#97]
This flag enables `requestAnimationFrame` method, which is a requirement for running React in the testing environment.

Currently I need to do the following:

```js
require('global-jsdom')(undefined, { pretendToBeVisual: true });
```

It would be great to make `global-jsdom/register` doing this by default.

**Comment by modosc** [comment]
thanks for the issue - i'm torn about changing this behavior by default since there'd be no easy way to opt out.

otoh - it would be nice for some variation of `import 'global-jsdom/register` / `-r global-jsdom/register
`  to support this out of the box. maybe a new entry point `global-jsdom/register-visual` ? i have to rethink the register code anyway due to #84 so i'll churn on this too.

**Comment by modosc** [comment]
i realized i'm already setting some default options _and_ making a major release with breaking changes so i went ahead and added this in too. i just published `8.0.0.beta2` with this change.

**Comment by just-boris** [comment]
Very nice, thank you!
